lib/topological_inventory/providers/common/operations/source.rb

```diff
@@ -123,6 +136,23 @@ def update_source(status)
           logger.availability_check("Failed to update Source id:#{source_id} - #{e.message}", :error)
         end
 
+        def update_source_by_kafka(status)
+          client = ManageIQ::Messaging::Client.open(:protocol => :Kafka, :host => messaging_host, :port => messaging_port)
```

Comment:
Is this set for the rdkafka library? I see v 1.0.0 is set in the gemspec.

Reply:
It should be - every one of our workers is updated to use it now I think.